en: add the doc about upgrading from 1.0 to 2.0 manually (#213)
pingcap/docs-dm: en/upgrade-dm-1.0.md

---

> title: Upgrade Data Migration

Reviewer: ```suggestion
title: Upgrade TiDB Data Migration Between 1.0.x Versions
```

> summary: Learn how to upgrade a Data Migration in 1.0.x version to an incompatible version.

Reviewer: ```suggestion
summary: Learn how to upgrade a TiDB DM cluster from a lower 1.0.x version to a higher 1.0.x version.
```

> aliases: ['/docs/tidb-data-migration/dev/dm-upgrade/','/en/tidb-data-migration/dev/dm-upgrade']

Reviewer: ```suggestion
aliases: ['/docs/tidb-data-migration/dev/dm-upgrade/','/tidb-data-migration/dev/dm-upgrade']
```
`en` should be ignored

---

> # Upgrade Data Migration in 1.0.x

Author: Shall I add a "v" here? @yikeke 
```suggestion
# Upgrade Data Migration in v1.0.x
```

Reviewer: ```suggestion
# Upgrade TiDB Data Migration Between 1.0.x Versions
```

> This document introduces how to upgrade your Data Migration (DM) in 1.0.x version to an incompatible version.

Reviewer: ```suggestion
This document introduces how to upgrade a TiDB DM cluster from a lower 1.0.x version to a higher 1.0.x version, and the major changes and other information about the upgraded version.
```
